src/Microsoft.DotNet.Darc/src/DarcLib.AzDev/IAzureDevOpsClient.cs

```diff
@@ -0,0 +1,126 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.DarcLib
+{
+    public interface IAzureDevOpsClient
+    {
+        /// <summary>
+        /// Edit the release Art
+        /// </summary>
+        /// <param name="accountName"></param>
+        /// <param name="projectName"></param>
+        /// <param name="releaseDefinition"></param>
+        /// <param name="build"></param>
+        /// <returns></returns>
+        Task<AzureDevOpsReleaseDefinition> AdjustReleasePipelineArtifactSourceAsync(string accountName, string projectName, AzureDevOpsReleaseDefinition releaseDefinition, AzureDevOpsBuild build);
+
+        /// <summary>
+        /// Deletes an Azure Artifacts feed and all of its packages
+        /// </summary>
+        /// <param name="accountName">Azure DevOps account name</param>
+        /// <param name="project">Project that the feed was created in</param>
+        /// <param name="feedIdentifier">Name or id of the feed</param>
+        /// <returns></returns>
+        Task DeleteFeedAsync(string accountName, string project, string feedIdentifier);
+
+        /// <summary>
+        /// Deletes a NuGet package version from a feed.
+        /// </summary>
+        /// <param name="accountName">Azure DevOps account name</param>
+        /// <param name="project">Project that the feed was created in</param>
+        /// <param name="feedIdentifier">Name or id of the feed</param>
+        /// <param name="packageName">Name of the package</param>
+        /// <param name="version">Version to delete</param>
+        /// <returns></returns>
```

Comment:
NIT: I think it would be cleaner if we deleted those empty tags.

Reply:
I'll do what we do in the rest of the code and return "async task"